static/app/views/replays/list/replaysErroneousDeadRageCards.tsx

```diff
@@ -0,0 +1,141 @@
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types';
+import EventView from 'sentry/utils/discover/eventView';
+import useReplayList from 'sentry/utils/replays/hooks/useReplayList';
+import {useHaveSelectedProjectsSentAnyReplayEvents} from 'sentry/utils/replays/hooks/useReplayOnboarding';
+import useOrganization from 'sentry/utils/useOrganization';
+import ReplayTable from 'sentry/views/replays/replayTable';
+import {ReplayColumn} from 'sentry/views/replays/replayTable/types';
+
+function ReplaysErroneousDeadRageCards() {
+  const location: Location = {
+    pathname: '',
+    search: '',
+    query: {},
+    hash: '',
+    state: '',
+    action: 'PUSH',
+    key: '',
+  };
+  const organization = useOrganization();
+
+  const eventViewErrors = EventView.fromNewQueryWithLocation(
```

Comment:
Question: This is using `fromNewQueryWithLocation` but the next variable on line 48 is using `fromSavedQuery`. Should they both be doing the same thing?

Reply:
yes definitely, i missed that! thank you